src/types/types.ts

```diff
@@ -16,13 +16,19 @@ export type List = Record<number, Cell>;
 
 export type Grid = List[];
 
+export enum ColumnWidthLocation {
+  Dimension = "dimensions",
+  Measures = "measures",
```

Comment:
Why not name it `PseudoDimensions`?

Reply:
I wanted this to reflect where in hyper cube the column width were stored. So measures means, `/qHyperCubeDef/qMeasures/${idx}/qDef/columnWidth`. `PseudoDimensions` isn't really a place where we can store values, it is just a placeholder for where the measures should be placed instead of dimension

Reply:
Sounds great 👍